src/tests/e2etest/HostNameTests.cs

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+using Microsoft.Azure.Devices.Shared;
+using Microsoft.Azure.Devices;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+
+namespace e2etesting
+{
+    [TestFixture]
+    public class HostNameTests : E2eTest
+    {   
+        string ComponentName = "HostName";
+        const int reposonseCodeSuccess = 200;
+        public partial class ExpectedDesiredProperty
+        {
+            public string value { get; set; }
+            public int ac { internal get; set; }
```

Comment:
Can we not just not include these properties and simply have them validated in the `E2eTest::AreEqualByJson`? This would make the desired property look just like the PnP object without these added ac/ad/av properties?

Reply:
I have removed ac/ad/av properties from the HostName class